src/scripts/main.js

```diff
@@ -22,7 +22,18 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.innerText = key;
+    ul.append(li);
+
+    createTree(li, data[key]);
```

Comment:
let's check if `data[key]` is not empty, to avoid empty `ul` in DOM
<img width="351" alt="image" src="https://user-images.githubusercontent.com/70220667/229190057-970429a2-647d-429d-9bcc-fbec4a244a26.png">